db/migrate/20200428220200_create_vita_partners.rb

```diff
@@ -0,0 +1,17 @@
+class CreateVitaPartners < ActiveRecord::Migration[6.0]
+  def change
+    create_table :vita_partners do |t|
+      t.string :name, null: false
+      t.string :display_name
+      t.string :logo_url
```

Comment:
might make sense to call this `logo_filename` or `logo_path` if it's just storing the name of the image file. I would generally expect a _url property to be a full URL